### PR TITLE
26 Test multithread response url usage

### PR DESCRIPTION
**Pull request by ivshapovalov**
#26

### CODE REVIEW

File: src/test/java/ua/com/juja/microservices/teams/slackbot/exceptions/ExceptionHandlerTest.java [hunk 1]
```diff
@@ -101,6 +106,99 @@ public void handleTeamExchangeException() throws Exception {
...
+        Thread.sleep(2000);
+
+        mvc.perform(MockMvcRequestBuilders.post(SlackUrlUtils.getUrlTemplate(teamsSlackbotActivateTeamUrl),
+                SlackUrlUtils.getUriVars("slashCommandToken", "/teams-activate",
+                        activateTeamCommandText1, responseUrl1))
+                .contentType(MediaType.APPLICATION_FORM_URLENCODED))
+                .andExpect(status().isOk())
+                .andExpect(content().string(ACTIVATE_TEAM_MESSAGE));
+
+        Boolean SecondThreadIsFinished = task.get();
```
Comment (petrokramar) [inline src/test/java/ua/com/juja/microservices/teams/slackbot/exceptions/ExceptionHandlerTest.java:175]: Variable SecondThreadIsFinished is never used. You can shorten this line to task.get().

**Review by petrokramar — APPROVED** [review]
Test works but a probability of such event is close to zero.

File: src/test/java/ua/com/juja/microservices/teams/slackbot/exceptions/ExceptionHandlerTest.java (hunk 1 above)
Comment (ivshapovalov) [reply src/test/java/ua/com/juja/microservices/teams/slackbot/exceptions/ExceptionHandlerTest.java:175]: This variable is only for understanding what happens.
"SecondThreadIsFinished" means that both threads finished and it is a time to verify all method invokations. Not earlier

Comment (petrokramar) [reply src/test/java/ua/com/juja/microservices/teams/slackbot/exceptions/ExceptionHandlerTest.java:175]: OK. I agree with you.

**Comment by OlgaKulikova** [comment]
@ivshapovalov please update the branch before merging.

**Comment by ivshapovalov** [comment]
Not needed yet

**Pull request closed without merging**